transcendence-app/src/chat/chatroom/infrastructure/db/postgres/chatroom.postgres.repository.ts

```diff
@@ -81,18 +83,43 @@ export class ChatroomPostgresRepository
       sort === BooleanString.True ? ChatroomKeys.NAME : ChatroomKeys.ID;
     const chatroomsData = await makeQuery<Chatroom>(this.pool, {
       text: `SELECT *
-      FROM ${this.table}
-      WHERE ${ChatroomKeys.NAME} ILIKE $1
-      ORDER BY ${orderBy}
-      LIMIT $2
-      OFFSET $3;`,
+             FROM ${this.table}
+             WHERE ${ChatroomKeys.NAME} ILIKE $1
+             ORDER BY ${orderBy}
+             LIMIT $2 OFFSET $3;`,
       values: [`%${search}%`, limit, offset],
     });
     return chatroomsData
       ? chatroomsData.map((chatroom) => new this.ctor(chatroom))
       : null;
   }
 
+  async getAuthUserPaginatedChatrooms(
+    authUserId: string,
+    queryDto: Required<PaginationWithSearchQueryDto>,
+  ): Promise<Chatroom[] | null> {
+    const { limit, offset, sort, search } = queryDto;
+    const orderBy =
+      sort === BooleanString.True
+        ? ChatroomMemberKeys.JOINED_AT
+        : ChatroomKeys.NAME;
+    const chatroomsData = await makeQuery<Chatroom>(this.pool, {
+      text: `SELECT *
```

Comment:
```suggestion
      text: `SELECT c.*
```

suggestion: We could select the chatroom columns because if we add more columns in the future, there could be some naming conflicts.